packages/web-components/src/components/button-group/button-group.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { customElement, html, LitElement } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
+import sameHeight from '@carbon/ibmdotcom-utilities/es/utilities/sameHeight/sameHeight';
+import styles from './button-group.scss';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+export enum BUTTON_TYPES {
+  /**
+   * Default / Primary
+   */
+  DEFAULT = 'primary',
+
+  /**
+   * Alternate / Tertiary
+   */
+  ALTERNATE = 'tertiary',
+}
+
+/**
+ * Button group.
+ *
+ * @element dds-button-group
+ */
+@customElement(`${ddsPrefix}-button-group`)
+class DDSButtonGroup extends LitElement {
+  static get stableSelector() {
+    return `${ddsPrefix}--button-group`;
+  }
+
+  static get buttonGroupItemSelector() {
+    return `${ddsPrefix}-button-group-item`;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  render() {
+    return html`
+      <ol>
+        <slot @slotchange="${this._handleSlotChange}" />
+      </ol>
```

Comment:
My apologies for slipping this through the crack in my earlier review. To [avoid duplicate element between the host and shadow DOM](https://github.com/carbon-design-system/ibm-dotcom-library/blob/v1.11.0-rc.0/packages/web-components/docs/coding-conventions.md#custom-element-itself-as-an-eleement). Please make sure adding `list` role in a similar manner as [`<dds-button-group-item>`](https://github.com/carbon-design-system/ibm-dotcom-library/blob/b502a3239c3497b6863bc8d99064890e3030208c/packages/web-components/src/components/button-group/button-group-item.ts#L29-L31). Also please apply the same style as `.bx--buttongroup` to the host. Thanks!

```suggestion
      <slot @slotchange="${this._handleSlotChange}" />
```

Reply:
@asudoh Just updated it, thanks for your reviews!